cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/aggregator/KafkaPartitionMetricSampleAggregatorTest.java

```diff
@@ -129,24 +129,24 @@ public void testAggregateWithUpdatedCluster() throws NotEnoughValidWindowsExcept
                                                          TOPIC0,
                                                          false,
                                                          Collections.singletonList(new MetadataResponse.PartitionMetadata(
-                                                             Errors.NONE, PARTITION, NODE_0,
+                                                             Errors.NONE, TP, Optional.of(NODE_0.id()),
                                                              Optional.of(RecordBatch.NO_PARTITION_LEADER_EPOCH),
-                                                             Arrays.asList(nodes()), Arrays.asList(nodes()),
+                                                             nodeIds(), nodeIds(),
                                                              Collections.emptyList()))));
     topicMetadata.add(new MetadataResponse.TopicMetadata(Errors.NONE,
                                                          TOPIC0 + "1",
                                                          false,
                                                          Collections.singletonList(new MetadataResponse.PartitionMetadata(
-                                                             Errors.NONE, 0, NODE_0,
+                                                             Errors.NONE, TP, Optional.of(NODE_0.id()),
```

Comment:
Here, not TP but tp1, because of the topic parameter is `TOPIC0+"1"`

Reply:
thanks